lib/src/generator/generator_utils.dart

```diff
@@ -34,6 +34,16 @@ String generateCategoryJson(Iterable<Categorization> categories, bool pretty) {
   return encoder.convert(indexItems.toList());
 }
 
+String removeHTMLTags(String? input) {
```

Comment:
Nit: spell HTML like "Html" in a variable name like this. https://dart.dev/guides/language/effective-dart/style#do-capitalize-acronyms-and-abbreviations-longer-than-two-letters-like-words